src/index.html

```diff
@@ -18,7 +18,7 @@
       </div>
       <div class="nav-bar__home-search">
         <div class="search-bar">
-          <label for="SearchInput" class="search-bar__search-label">Search</label>
+          <label for="searchBarInput" class="search-bar__search-label">Search</label>
           <input type="search" class="search-bar__search-input" name="SearchInput" value="" id="searchBarInput">
           <button type="button" class="search-bar__initiate-search-button">
```

Comment:
We can delete this button since our user does not need to click it to search, but let's keep the span with the search icon.